src/main/resources/internal-admin-api.yaml

```diff
@@ -1595,6 +1597,13 @@ definitions:
         type: string
         description: The redirection URI for login
 
+  PasswordResetValidation:
+    type: object
+    properties:
+      urlParameters:
```

Comment:
Think this could be more descriptive - maybe "initialQueryParameters" and a description along the lines of "the initial query parameters used when submitting the request, to allow for resubmission after expiry"